src/AWS.Deploy.Orchestration/ServiceHandlers/AWSElasticBeanstalkHandler.cs

```diff
@@ -204,6 +212,62 @@ public void SetupWindowsDeploymentManifest(Recommendation recommendation, string
             }
         }
 
+        /// <summary>
```

Comment:
Understandable why it'd be difficult to share implementations, but might be nice to add a note here about keeping this in sync with the CDK implementation.

Reply:
Added a comment.